src/plugins/discover/public/application/components/sidebar/discover_field.tsx

```diff
@@ -63,6 +63,10 @@ export interface DiscoverFieldProps {
    * @param eventName
    */
   trackUiMetric?: (metricType: UiStatsMetricType, eventName: string | string[]) => void;
+  /**
+   * Shows Add button at all times and not only on focus
+   */
+  mobile?: boolean;
```

Comment:
Can we name this so that it corresponds to functionality more? Like `showAddButton` or `alwaysShowAddButton`? Or even better - just pass in the appropriate className?

Reply:
sure, since it would add a CSS class to a specific element of this component, I would prefer `alwaysShowAddButton`, it's clearer that having to pass a className in the container of this component

Reply:
will implement `alwaysShowAddButton ` ... tomorrow